tools/RAiDER/cli/raiderDelay.yaml

```diff
@@ -0,0 +1,60 @@
+# vim: set filetype=yaml:
+##------------------------ raiderDelay.cfg ------------------------##
+########## 1. Date and time
+## Dates can be a single date, two dates that define a set of dates every day, or two dates with an interval
+## Standard formats are accepted. 
+date_start:   # first date to download. Specified as YYYYMMDD
+date_end:   # last date to download, auto for None (single date)
+date_step:   # date interval to download; e.g. 12 for every 12 days. Auto for none (downloads every day)
+date_list:   # Pass in a list of dates directly to process
+time:   # time to download in UTC time. Specified as HH:MM:SS or HH:MM
+
+########## 2. Weather model
+## Currently implemented weather models include: 
+## ERA-5 (ECMWF), ERA-5T (ECMWF), HRES (ECMWF), ERA-I (ECMWF), NCMR (ISRO), HRRR (NOAA), GMAO (NASA), MERRA-2 (NASA)
+weather_model:   ## don't include the text in parentheses in the models above
+
+########## 3. Query points
+## There are four options for specifying query points: 
+## 1. lat/lon raster files (such as produced by the ISCE software)
```

Comment:
I think now this gets changed with @piyushrpt thoughts. In the end we are just looking for a GDAL compatible file no, instead of ISCE compatible?

Reply:
These isce files are mentioned only as an example of radar-coordinate files, doesn't have to be isce-compatible.